spec/features/user_views_all_books_spec.rb

```diff
@@ -10,4 +10,16 @@
     expect(page).to have_content("Practical Vim")
     expect(page).to have_content("The Giver")
   end
+
+  scenario "by pagination" do
+    create_list(:book, Book.paginates_per(2) + 1)
+
+    visit books_path
+
+    expect(page).to have_pagination
+  end
+
+  def have_pagination
+    have_css(".pagination")
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.